mod_wsbridge/mod_wsbridge.c

```diff
@@ -384,6 +450,88 @@ websocket_write_back(struct lws *wsi_in, enum lws_write_protocol type, char *str
 	return n;
 }
 
+void add_content_type(private_t *tech_pvt, cJSON* json) {
+	if (cJSON_GetObjectItem(json, "content-type")) {
+		cJSON_DeleteItemFromObject(json, "content-type");
+	}
+	cJSON_AddItemToObject(json, "content-type", cJSON_CreateString(tech_pvt->content_type));
+}
+
+int is_mute_event(char* event, char* method) {
+	return !strcmp(event, "websocket:media:update") && !strcmp(method, "update");
+}
+
+void reset_ws_write_indexes(private_t *tech_pvt) {
+	switch_mutex_lock(tech_pvt->write_mutex);
+	tech_pvt->write_count = 0;
+	tech_pvt->write_start = 0;
+	tech_pvt->write_end = 0;
+	switch_mutex_unlock(tech_pvt->write_mutex);
+}
+
+char* get_valuestring(cJSON* json, char* field) {
+	char* valuestring = NULL;
+	cJSON* json_field = NULL;
+	json_field = cJSON_GetObjectItem(json, field);
+	if (json_field) {
+		valuestring = json_field->valuestring;
+	}
+	return valuestring;
+}
+
+void on_event(private_t *tech_pvt, cJSON* json) {
+	char* event = NULL;
+	char* method = NULL;
+	event = get_valuestring(json, "event");
+	method =  get_valuestring(json, "method");
+
+	if (!event || !method) {
+		return;
+	}
+
+	if (is_mute_event(event, method)) {
```

Comment:
Could we add a configuration flag in here so that we can enable the additional frames enabled or not? It could be a config flag through the XML and could be hot-loaded.

Reply:
added a config flag in wsbridge.conf.xml to enable/disable the transmissions of control events to the ws bridge.
`<param name="enable_control_events" value="1"/>`